concurrent-tck/pom.xml

```diff
@@ -148,14 +154,25 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.2</version>
+                <version>3.2.5</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.apache.maven.surefire</groupId>
+                        <artifactId>surefire-junit-platform</artifactId>
+                        <version>3.2.5</version>
+                    </dependency>
+                </dependencies>
                 <configuration>
                     <systemPropertyVariables>
                     </systemPropertyVariables>
-                    <suiteXmlFiles>
-                        <suiteXmlFile>${tck-suite.xml}</suiteXmlFile>
-                    </suiteXmlFiles>
-                    <testSourceDirectory>${basedir}${file.separarator}src${file.separarator}main${file.separarator}java${file.separarator}</testSourceDirectory>
+                    <groups>full</groups>                        <!-- Groups to include i.e. web/full -->
+                    <!-- <includes>
+                        <include>ee.jakarta.tck.concurrent.spec.ManagedThreadFactory.injected.*Tests</include>
+                    </includes> -->
```

Comment:
Why is this commented out? An explanation in the pom wouldn't go amiss.

Reply:
I am adding a comment; this is useful when debugging some particular tests. I would definitely forgot, how to write it :-)